<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PACSG1957NLN" failures="0" tests="3" name="Hello7.day4" time="0.009" errors="1" timestamp="2024-12-02T16:43:21 SGT" skipped="0">
  <testcase classname="Hello7.day4" name="WebloginHomeLoan" time="0.000">
    <error message="
Parameter &#039;URL&#039; is required by @Test on method WebloginHomeLoan but has not been marked @Optional or defined
in C:\Users\vincent.ong\AppData\Local\Temp\testng-eclipse--307692246\testng-customsuite.xml" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Parameter 'URL' is required by @Test on method WebloginHomeLoan but has not been marked @Optional or defined
in C:\Users\vincent.ong\AppData\Local\Temp\testng-eclipse--307692246\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:264)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:352)
at org.testng.internal.Parameters.createParameters(Parameters.java:690)
at org.testng.internal.Parameters.handleParameters(Parameters.java:885)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- WebloginHomeLoan -->
  <testcase classname="Hello7.day4" name="LoginAPIHomeLoan" time="0.008"/>
  <testcase classname="Hello7.day4" name="MobileLoginHomeLoan" time="0.001"/>
</testsuite> <!-- Hello7.day4 -->
